### PR TITLE
Use fallback values of env keys

### LINKED ISSUE #461: Add default values to Instant Meilisearch's client
**Description**
The Interactive Search component [on the homepage](https://www.meilisearch.com/) allows us to display and search inside movies: 

Capture d’écran 2022-09-13 à 09 13 57

To do so, we use [Instant Meilisearch](https://github.com/meilisearch/instant-meilisearch). The client needs to be initialized with a host and an apiKey (`NEXT_PUBLIC_DEMO_MOVIES_HOST` and `NEXT_PUBLIC_DEMO_MOVIES_API_KEY`).

By running the website locally, these environment variables are loaded thanks to a `.env` file, but if they are not provided, the website crashes on `yarn dev`:

Capture d’écran 2022-09-13 à 09 17 33

We should provide a default value to these environment variables. I suggest `http://0.0.0.0:7700` as a host, and `searchKey` for the apiKey.

The changes should be made inside the `src/components/InteractiveSearch.js` component, inside the `.env.example`, and also inside the `src/stories/Searchbox.stories.js` file as well.

### PR DESCRIPTION
**Pull request by arifszn**
# Pull Request

## What does this PR do?
Fixes #461

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
Hey @arifszn thanks a lot for you contribution! 🔥

For readability purposes, can you move the variable declaration to the top of the file the following way:

After the imports: 

```js
const NEXT_PUBLIC_DEMO_MOVIES_HOST = process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST
  ? process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST
  : 'http://0.0.0.0:7700'
const NEXT_PUBLIC_DEMO_MOVIES_API_KEY = process.env
  .NEXT_PUBLIC_DEMO_MOVIES_API_KEY
  ? process.env.NEXT_PUBLIC_DEMO_MOVIES_API_KEY
  : 'masterKey'
```

in the client: 
```js
instantMeiliSearch(
        NEXT_PUBLIC_DEMO_MOVIES_HOST,
        NEXT_PUBLIC_DEMO_MOVIES_API_KEY,
        {
          paginationTotalHits: 4,
          primaryKey: 'id',
        }
```

In both files! In `InteractiveSearch` add it after the export. 

Do not forget to run the linter `yarn lint` to ensure the styling is correct

**Comment by arifszn** [comment]
Done. Please review it now.

**Comment by bidoubiwa** [comment]
Hey @arifszn thanks for the update. We will review it when we have time :)

File: src/components/InteractiveSearch.js [hunk 1]
```diff
@@ -9,6 +9,14 @@ import Typography from 'components/Typography'
 import Lottie from 'components/Lottie'
 import { useInView } from 'react-intersection-observer'
 
+const NEXT_PUBLIC_DEMO_MOVIES_HOST = process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST
+  ? process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST
+  : 'http://0.0.0.0:7700'
```
Comment (mdubus) [inline src/components/InteractiveSearch.js:14]: You can make it more readable by doing so:

```
const NEXT_PUBLIC_DEMO_MOVIES_HOST = process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST ||
'http://0.0.0.0:7700'
```

File: src/components/InteractiveSearch.js [hunk 2]
```diff
@@ -9,6 +9,14 @@ import Typography from 'components/Typography'
...
+const NEXT_PUBLIC_DEMO_MOVIES_API_KEY = process.env
+  .NEXT_PUBLIC_DEMO_MOVIES_API_KEY
+  ? process.env.NEXT_PUBLIC_DEMO_MOVIES_API_KEY
+  : 'searchKey'
```
Comment (mdubus) [inline src/components/InteractiveSearch.js:18]: Same as above :)

File: src/stories/Searchbox.stories.js [hunk 3]
```diff
@@ -3,6 +3,14 @@ import { instantMeiliSearch } from '@meilisearch/instant-meilisearch'
 import Searchbox from 'components/Searchbox'
 import { InstantSearch } from 'react-instantsearch-dom'
 
+const NEXT_PUBLIC_DEMO_MOVIES_HOST = process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST
+  ? process.env.NEXT_PUBLIC_DEMO_MOVIES_HOST
+  : 'http://0.0.0.0:7700'
+const NEXT_PUBLIC_DEMO_MOVIES_API_KEY = process.env
+  .NEXT_PUBLIC_DEMO_MOVIES_API_KEY
+  ? process.env.NEXT_PUBLIC_DEMO_MOVIES_API_KEY
+  : 'searchKey'
```
Comment (mdubus) [inline src/stories/Searchbox.stories.js:12]: Same as above :)

**Review by mdubus — CHANGES_REQUESTED** [review]
Hello @arifszn 👋 

Thanks a lot for your PR 🙏 There are small things to change to improve readability and then we're all good 🥳 

Have a nice day! ✨

File: src/components/InteractiveSearch.js (hunk 1 above)
Comment (arifszn) [reply src/components/InteractiveSearch.js]: Done.

**Review by mdubus — APPROVED** [review]
That's perfect, thanks a lot for your changes! 🥳

**Pull request merged**